web/beacon-app/src/features/topics/api/topicDetailApiService.ts

```diff
@@ -0,0 +1,15 @@
+import type { ApiAdapters } from '@/application/api/ApiAdapters';
+import type { Request } from '@/application/api/ApiService';
+import { getValidApiResponse } from '@/application/api/ApiService';
+import { APP_ROUTE } from '@/constants';
+
+import { Topic } from '../types/topicService';
+export function topicRequest(request: Request): ApiAdapters['getTopic'] {
+  return async (topicsID: string) => {
```

Comment:
```suggestion
  return async (topicID: string) => {
```

Reply:
This suggestion was made since there's only one topic ID. If this is changed, I believe this will need to be changed elsewhere.

Reply:
Thanks. I forgot to change the params since I took it from topicServieAPI.